packages/ic-asset-certification/src/asset_router.rs

```diff
@@ -259,9 +259,6 @@ impl<'content> AssetRouter<'content> {
     /// Certifies multiple assets and inserts them into the router, to be served
     /// later by the [serve_asset](AssetRouter::serve_asset) method.
     ///
-    /// The assets are certified using the provided
```

Comment:
Small left over from a previous iteration that required consumers to pass the `HttpCertificationTree` as an argument to this function.